### PR TITLE
Prl 6200 dac duplicate ids

### PR DESCRIPTION
**Pull request by BharadwajSairam**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/PRL-6200

### Change description ###

DAC - Duplicate IDs/DAC_Duplicate_IDs_01

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/steps/common/form/form.njk [hunk 1]
```diff
@@ -23,7 +23,7 @@
             value: true,
             preventDoubleClick: true,
             attributes: {
-              id: 'main-form-submit'
```
Comment (vivek-sekhar) [inline src/main/steps/common/form/form.njk:26]: Is the id used anywhere in the code? Is the change tested and working without breaking any functionality.

File: src/main/steps/common/form/form.njk [hunk 2]
```diff
@@ -93,7 +93,7 @@
             name: "onlycontinue",
             value: true,
             attributes: {
-              id: 'main-form-save-as-draft'
+              id: 'main-form-only-continue'
```
Comment (vivek-sekhar) [inline src/main/steps/common/form/form.njk:96]: Is the id used anywhere in the code? Is the change tested and working without breaking any functionality.

File: src/main/steps/common/respondent-summary/form.njk [hunk 3]
```diff
@@ -23,37 +23,9 @@
...
-          <div class="govuk-checkboxes govuk-!-padding-bottom-2">
-            <div class="govuk-checkboxes__item">
-              <input class="govuk-checkboxes__input" style="background-color: white" id="declarationCheck" name="declarationCheck" type="checkbox" value="declaration">
-              <label class="govuk-label govuk-checkboxes__label" for="declarationCheck">
-                {{believeFacts}}
-              </label>
-            </div>
-          </div>
-        </div>
+
```
Comment (vivek-sekhar) [inline src/main/steps/common/respondent-summary/form.njk:28]: Is the change tested and working, if so please add evidence in the ticket along with other evidences.

File: src/main/steps/tasklistresponse/summary/content.ts [hunk 4]
```diff
@@ -723,12 +724,32 @@ const cy: typeof en = (content: CommonContent) => {
...
+    },
+    submit: {
+      type: 'inset',
+      label: l => `<p class="govuk-body">${l.submit}</p><p class="govuk-body">${l.download}</p>`,
+    },
+    statementOfTruthSubmission: {
+      type: 'textAndHtml',
+      label: l => `${l.statementOfTruthSubmission}`,
+    },
     declarationCheck: {
```
Comment (vivek-sekhar) [inline src/main/steps/tasklistresponse/summary/content.ts:747]: Is this working as expected? tested the changes?

**Review by vivek-sekhar — CHANGES_REQUESTED** [review]

File: src/main/steps/tasklistresponse/summary/content.ts (hunk 4 above)
Comment (BharadwajSairam) [reply src/main/steps/tasklistresponse/summary/content.ts:747]: Yes tested, attached test evidence in the ticket

File: src/main/steps/common/respondent-summary/form.njk (hunk 3 above)
Comment (BharadwajSairam) [reply src/main/steps/common/respondent-summary/form.njk:28]: Yes tested, attached test evidence in the ticket

File: src/main/steps/common/form/form.njk (hunk 2 above)
Comment (BharadwajSairam) [reply src/main/steps/common/form/form.njk:96]: Not used anywhere else in application, no breaking functionality observed during testing

File: src/main/steps/common/form/form.njk (hunk 1 above)
Comment (BharadwajSairam) [reply src/main/steps/common/form/form.njk:26]: Not used anywhere else in application, no breaking functionality observed during testing

Comment (BharadwajSairam) [reply src/main/steps/common/form/form.njk]: Reverted this change, its already under conditional, based on accesscodecheck - no need to change the id for this field

**Pull request merged**